source/new-starter/prototype/index.html.md.erb

```diff
@@ -7,17 +7,13 @@ review_in: 6 months
 
 # Set up the GOV.UK account manager prototype
 
-The GOV.UK account manager prototype is an application to test how users:
-
-- authenticate their data
-- authorise their data for exchange
-- stay informed of the use of their data
-- manage consent for use of their data
+The GOV.UK account manager prototype and attribute store are a central part of GOV.UK Accounts.
+You can find out more about how they work with other services in [How GOV.UK Accounts works](/new-starter/how-it-works/#how-gov-uk-accounts-works).
```

Comment:
Should we be using relative or absolute routes?

Reply:
relative